CHANGELOG.md

```diff
@@ -4,6 +4,17 @@ All notable changes to the "vscode-java-debugger" extension will be documented i
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+
+## 0.8.0 - 2018-4-3
```

Comment:
wait for build succeeded before launch debug session (#257) 
fix the invalid stack exception during evaluation (#163)